pkg/occlient/occlient.go

```diff
@@ -876,6 +910,81 @@ func (c *Client) UpdateBuildConfig(buildConfigName string, projectName string, g
 	return nil
 }
 
+// AddSupervisorToDC adds the supervisor to the deployment config
```

Comment:
I'm not sure that this is the correct name for this function. It doesn't add a supervisor to the deployment config. The supervisor is added to the image via BuildConfig, this just adds initContainer and volume that is needed for it work properly

Reply:
@kadel Can you suggest any name? I am confused